igneous/tasks/skeleton.py

```diff
@@ -448,16 +452,30 @@ def get_unfused(self, labels, filenames, cv):
       n_blocks = max(len(filenames) // block_size, 1)
       blocks = sip(filenames, block_size)
 
+    if frag_path:
```

Comment:
I feel like this could be written more concisely.

```
frag_prefix = cv.cloudpath
local_input = False
if frag_path:
   frag_prefix = frag_path
if frag_prefix.startswith("file://"):
   local_input = True
   frag_prefix = frag_prefix.replace("file://", "", 1)
```

Reply:
So there is no risk not using `cv.meta.path.protocol == "file"` to determine the protocol, when should that be used?

Reply:
Do you want me to do the same for the mesh code as well?

Reply:
At this time, I think it is fine. This is because CloudVolume doesn't allow implied file paths, you have to specify `file://` every time. Which might not be ideal... probably another way to do it that is more robust might be to use path.extract since I'll have to update that if I ever allow simplified paths.

Reply:
Yes, if you update this code, update both.